src/benchmarks/micro/libraries/System.Collections/Sort.cs

```diff
@@ -108,6 +345,7 @@ private sealed class ComparableComparerClass : IComparer<T>
 
         private readonly struct ComparableComparerStruct : IComparer<T>
         {
+            [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Why can't JIT inline it? This is tiny method, it does not sound right to require `AggressiveInlining` on it for good performance.

Reply:
(multiple places)

Reply:
will remove. old habits 🙄